CHANGELOG.md

```diff
@@ -8,6 +8,18 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 For compatibility information see `govukFrontendVersion` and `hmrcFrontendVersion` in
 [LibDependencies](project/LibDependencies.scala)
 
+## [7.11.0] - 2023-06-13
+
+### Changed
+
+- Restore previous behaviour of `mainContentLayout` and `pageLayout` in
```

Comment:
@ellamdav would it be ok to say something like "in a previous version {version} the semantics of passing None changed unintentionally from causing no layout to be applied, to falling back to applying the default layouts. If you pass None to either of these parameters, please double check your usage after upgrading to this version to ensure there are no unexpected layout changes as a result."